pkg/services/auth/auth_token_test.go

```diff
@@ -60,8 +60,18 @@ func TestUserAuthToken(t *testing.T) {
 				So(userToken, ShouldBeNil)
 			})
 
+			Convey("soft revoking existing token should not delete token", func() {
```

Comment:
Nit, but I think this reads more clearly:
```suggestion
			Convey("soft revoking existing token should not delete it", func() {
```